### PR TITLE
Clear In-Memory Redux Store on Logout

### LINKED ISSUE #132: Clear Redux Store on Logout
[Project/Issue Tracker](https://github.com/Code-4-Community/lucys-love-bus-frontend/projects/1)

### Objective

Some pieces of data are user specific (think contacts) and arent cleared on logout, which creates fun bugs like being able to see the info of the person who last logged in (yikes)!
 
### Tech Spec
- Logout fully clears redux store

### PR DESCRIPTION
**Pull request by rymaju**
## Issue

Closes #132 

## Changes
- Force refresh on logout to clear in-memory redux store, localstorage redux store is already cleared on logout

### CODE REVIEW

**Review by anirudh723 — APPROVED** [review]
🚢

File: src/components/navbar/index.tsx [hunk 1]
```diff
@@ -131,6 +131,7 @@ const NavBar: React.FC<NavBarProps> = ({ tokens }) => {
           if (asyncRequestIsComplete(tokens)) {
             dispatch(logout());
             history.push(Routes.HOME);
+            history.go(0);
```
Comment (vrautela) [inline src/components/navbar/index.tsx:134]: ❓ What is this doing? I'm not super familiar with how the history object but I read [https://reactrouter.com/web/api/history](url) and it seems like `history.go(0)` would move the history object back 0 times.

Comment (rymaju) [reply src/components/navbar/index.tsx:134]: `history.go(0)` esentially just lets reloads the page

Comment (onb6) [reply src/components/navbar/index.tsx:134]: oh interesting this is super useful to know

**Review by onb6 — APPROVED** [review]
Nice!

**Pull request merged**